src/sagemaker/workflow/step_collections.py

```diff
@@ -250,32 +254,32 @@ def __init__(
         self,
         name: str,
         estimator: EstimatorBase,
-        model_data,
-        model_inputs,
-        instance_count,
-        instance_type,
-        transform_inputs,
-        description: str = None,
-        display_name: str = None,
+        model_data: str,
+        model_inputs: CreateModelInput,
+        instance_count: int,
+        instance_type: str,
+        transform_inputs: TransformInput,
+        description: Optional[str] = None,
+        display_name: Optional[str] = None,
         # model arguments
-        image_uri=None,
-        predictor_cls=None,
-        env=None,
+        image_uri: Optional[str] = None,
+        predictor_cls: Optional[Callable[[str, Session], Predictor]] = None,
```

Comment:
What is this for?

Reply:
predictor is just a wrapper around an endpoint, which handles the serialization and deserialization of the request and response to and from an endpoint.